cardano-ledger-core/src/Cardano/Ledger/BaseTypes.hs

```diff
@@ -581,3 +603,7 @@ instance FromCBOR Network where
     word8ToNetwork <$> fromCBOR >>= \case
       Nothing -> cborError $ DecoderErrorCustom "Network" "Unknown network id"
       Just n -> pure n
+
+isSNothing :: StrictMaybe a -> Bool
```

Comment:
We should add this to `strict-containers`, but I agree it's better here for now!

Reply:
That was exactly what I thought to myself as well. However, I do need to get access to cardano-base first

Reply:
there's gotta be a "no es nada" joke here somewhere.. :)